examples/simple-platform/run.md

```diff
@@ -0,0 +1,177 @@
+# Running the Platform Example Locally
+
+## Setup
+
+### Create a GitHub development application for the `account-app` OAuth process
+
+The accounts app supports account creation and authentication via GitHub only. 
+In order to run a user authentication loop, you need to create a new GitHub 
+OAuth app and configure it to connect to the `web-auth-endpoint` node.
+
+1. Go to [github.com](https://github.com) and log into your account.
+
+1. Navigate to Settings > Developer Settings > OAuth Apps.
+
+1. Click the **New OAuth App** button to create a new OAuth app and configure 
+it.
+
+1. In the **Application name** field, give the app a name of your choosing, 
+e.g. `Interbit Test App`.
+
+1. Fill in the **Homepage URL** field with any URL. Any URL will do since we 
+are running the example locally.
+
+1. Fill in the **Authorization callback URL** field with 
+`http://localhost:8888/oauth/github`.
+
+   Note: The callback URL is case sensitive and the port specified is for 
+running the `web-auth-endpoint` node locally.
+
+1.  Click the **Register application** to finish creating the OAuth app.
+
+1. Leave this browser tab open as we will copy values from here in the steps 
+below. 
+
+
+### Generate a set of private and public keys for `platform-deploy` and `web-auth-endpoint`
```

Comment:
What are these things, `platform-deploy` and `web-auth-endpoint`? In `structure.md` we call them hypervisors. We should do so here too.